html/config/package.md

```diff
@@ -0,0 +1,14 @@
+macOS
+Stable Nightly
+brew Pkg.jil source
+C++ Python Julia All
+
+Linux (Debian)
+Stable Nightly
+apt pip conda Pkg.jil source
+C++ Python Julia All
```

Comment:
So, I'm not sure about the difference between the language and the package manager, since sometimes a package manager can imply a language.  For instance, it's not possible to install mlpack for C++ through Julia's `Pkg.jl`.  So, I wonder if maybe it is better to just have one "package manager" row and have it say, e.g., for Linux,
```
"apt (Debian/Ubuntu)" "dnf (Fedora/RHEL)" "pacman (Arch)" "conda (Python)" "pip (Python)" "Pkg.jl (Julia)" "CRAN (R)" "source"
```
or something more like that?  Let me know what you think.

Reply:
Also, do you think we should add "CLI" as a language here?

Reply:
I like the idea, will add it to the issue.